timemachine/cpp/src/nonbonded_common.cu

```diff
@@ -0,0 +1,127 @@
+#include <set>
```

Comment:
why is this a `.cu` file?

Reply:
// includes CUDA Runtime
#include <cuda_runtime.h>

// maybe you need also helpers
#include <helper_cuda.h>
#include <helper_functions.h> // helper utility functions

Reply:
^ does this work? (Sorry to nit, but trying to keep scope of .cu files somewhat minimal), and I probably didn't do this consistently for previous files either.

Reply:
Will give that a shot and get back to you.

Reply:
Included `<cuda_runtime.h>` in the neighborlist header in https://github.com/proteneer/timemachine/pull/1053/commits/0e617ed927eb592767a67f75c127ca6b322b6af0

Was that we were ending up at the neighborlist with gcc rather than nvcc. Good call to do this